tarodb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -224,8 +224,7 @@ CREATE TABLE IF NOT EXISTS asset_minting_batches (
 
     minting_tx_psbt BLOB,
 
-    -- TODO(roasbeef): redundant w/ info in genesis_assets table?
-    minting_output_index SMALLINT,
```

Comment:
Yeah I think originally this was meant to track the minting output, but then that information is available indirectly via either the PSBT or the genesis assets table.